Vungle Sample App Swift/Vungle Sample App Swift/ViewController.swift

```diff
@@ -226,30 +156,96 @@ class ViewController: UIViewController, VungleSDKDelegate {
+        else if (placementID == kVungleTestPlacementID02) {
+            playButton2.updateState(enabled: isAdPlayable)
+            loadButton2.updateState(enabled: !isAdPlayable)
+        }
+        else if (placementID == kVungleTestPlacementID03) {
+            playButton3.updateState(enabled: isAdPlayable)
+            loadButton3.updateState(enabled: !isAdPlayable)
+        }
+    }
+    
+    func vungleWillShowAd(forPlacementID placementID: String?) {
+        print("-->> Delegate Callback: vungleSDKwillShowAd")
+        if (placementID == kVungleTestPlacementID01) {
+            print("-->> Ad will show for Placement 01")
+            playButton1.updateState(enabled: false)
+            
+        }
+        else if (placementID == kVungleTestPlacementID02) {
+            print("-->> Ad will show for Placement 02")
+            playButton2.updateState(enabled: false)
+        }
+        else if (placementID == kVungleTestPlacementID03) {
+            NSLog("-->> Ad will show for Placement 03")
+            playButton3.updateState(enabled: false)
+        }
+    }
+    
+    func vungleWillCloseAd(with info: VungleViewInfo, placementID: String) {
+        print("-->> Delegate Callback: vungleWillCloseAdWithViewInfo")
+        if (placementID == kVungleTestPlacementID01) {
+            print("-->> Ad is closed for Placement 01")
+        }
+        else if (placementID == kVungleTestPlacementID02) {
+            print("-->> Ad is closed for Placement 02")
+        }
+        else if (placementID == kVungleTestPlacementID03) {
+            print("-->> Ad is closed for Placement 03")
+        }
+        
+        print("Info about ad viewed: \(info)")
+        
+        updateButtons()
+        sdk.muted = false
+    }
     
     private func updateButtons () {
-        self.updateButtonState(for: self.playButton1, enabled: self.sdk.isAdCached(forPlacementID: kVungleTestPlacementID01) ? true:false)
-        self.updateButtonState(for: self.playButton2, enabled: self.sdk.isAdCached(forPlacementID: kVungleTestPlacementID02) ? true:false)
-        self.updateButtonState(for: self.loadButton2, enabled: self.sdk.isAdCached(forPlacementID: kVungleTestPlacementID02) ? false:true)
-        self.updateButtonState(for: self.playButton3, enabled: self.sdk.isAdCached(forPlacementID: kVungleTestPlacementID03) ? true:false)
-        self.updateButtonState(for: self.loadButton3, enabled: self.sdk.isAdCached(forPlacementID: kVungleTestPlacementID03) ? false:true)
+        playButton1.updateState(enabled: sdk.isAdCached(forPlacementID: kVungleTestPlacementID01))
+        playButton2.updateState(enabled: sdk.isAdCached(forPlacementID: kVungleTestPlacementID02))
+        loadButton2.updateState(enabled: !sdk.isAdCached(forPlacementID: kVungleTestPlacementID02))
+        playButton3.updateState(enabled: sdk.isAdCached(forPlacementID: kVungleTestPlacementID03))
+        loadButton3.updateState(enabled: !sdk.isAdCached(forPlacementID: kVungleTestPlacementID03))
```

Comment:
I think this makes things a little tighter. I don't think you need the tertiary operator if you're just returning `true:false` since that's what the method does anyway